### PR TITLE
Getting the project to run locally

### PR DESCRIPTION
**Pull request by allenhsieh**
1. Separated server local into server.js and app.js
2. Ripped out mongoose model into it's own file
3. Rewrote seed generation script (script was previously broken)
4. Ran initial seed of 10k records and confirmed component was running on local host

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,28 +1,18 @@
...
-const dbUtils = require('../database/dbUtils');
-const Models = require('../database/models')
+const mongoose = require('mongoose');
+const app = require('./app.js');
 
-app.use(morgan('dev'));
-app.use(express.static(path.join(__dirname, '../public')));
-app.use('/:id', express.static(path.join(__dirname, '../public')));
-app.use(cors());
+mongoose.connect((process.env.MONGODB_URI || 'mongodb://localhost/galleries'), { useCreateIndex: true, useNewUrlParser: true });
```
Comment (cjm771) [inline server/server.js]: db.js is using `localhost/gallery`.  is this file not being used anymore? if so remove `db.js`

**Comment by cjm771** [comment]
endpoints in readme , not sure what they are without diving into code. would be useful.

**Comment by cjm771** [comment]
tests exist?

**Comment by allenhsieh** [comment]
> basic readme changes and you good.

Talked about in slack but the README currently in the repo is inherited from FEC. Opened up new issue to tackle README re-write at later date #6

**Comment by allenhsieh** [comment]
> tests exist?

Good call, looking through it seems like jonathan wasn't able to create tests for FEC. I'll add another issue #7 

The way we handled test was just a unit test for routes (why we split the server code to app+server)

**Pull request merged**